metron-analytics/metron-statistics/src/main/java/org/apache/metron/statistics/StellarStatisticsFunctions.java

```diff
@@ -141,8 +141,21 @@ public Object apply(List<Object> args) {
 
       // add each of the numeric values
       for(int i=1; i<args.size(); i++) {
-        double value = convert(args.get(i), Double.class);
-        stats.addValue(value);
+        Object n = args.get(i);
+        if( n != null) {
+          if(n instanceof Iterable) {
+            for(Object num : (Iterable<Object>)n) {
+              if(num != null) {
```

Comment:
Just a nit, but why 'double' and not Double?
Is there a reason to box?

Reply:
No good reason :)